WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -199,17 +223,23 @@ final class NewOrderViewModel: ObservableObject {
     private let orderSynchronizer: OrderSynchronizer
 
     init(siteID: Int64,
+         initialOrder: Order? = nil,
          stores: StoresManager = ServiceLocator.stores,
          storageManager: StorageManagerType = ServiceLocator.storageManager,
          currencySettings: CurrencySettings = ServiceLocator.currencySettings,
          analytics: Analytics = ServiceLocator.analytics,
          featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
         self.siteID = siteID
+        if let initialOrder = initialOrder {
+            self.flow = .editing(initialOrder: initialOrder)
+        } else {
+            self.flow = .creation
+        }
```

Comment:
Do you think it will be easier/better if we let the consumer decide which flow to use?
I mean pass the `flow` property in the init?

Reply:
Looks better! But required a bit of `if case let` dance :)
5102e04